src/main/java/com/github/manolo8/darkbot/backpage/HangarManager.java

```diff
@@ -49,8 +55,7 @@ public boolean changeHangar(String hangarID) {
     }
 
     public Boolean checkDrones() {
-        updateHangars();
-        updateDrones();
+        updateHangarData();
```

Comment:
What will happen if list of hangars hasn't been updated? It won't know active hangar id and won't be able to fetch updated drones